GroceryStore/GroceryStore.Models/Product.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GroceryStore.Constants;
+
+
+
+namespace GroceryStore.Models
+{
+    public class Product
+    {
+        public string Name { get; set; }
+        public ProductCategories.Category Category { get; set; }
+        public double Price { get; set; }
+        public int Amount { get; set; }
```

Comment:
Mmm I think that Amount property should not be in the class. Because e.g. you create Product "Tomato". And customer added three tomatoes to his cart. So customer know how much tomatoes in his cart. But why does the tomato know how much of it is in the cart?)))